pages/linux/systemctl.md

```diff
@@ -3,9 +3,10 @@
 > Control the systemd system and service manager.
```

Comment:
Welcome to tldr @levid0s 
Can you please sign the CLA

Reply:
@levid0s Can you check this out?